content/webpack.config.js

```diff
@@ -2,10 +2,12 @@ const autoprefixer = require('autoprefixer');
 const CleanWebpackPlugin = require('clean-webpack-plugin');
 const path = require('path');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const FixStyleOnlyEntriesPlugin = require("webpack-fix-style-only-entries");
 
 module.exports = {
     entry: {
-        bundle: './Assets/Scripts/index.js'
+        bundle: './Assets/Scripts/index.js',
```

Comment:
I think "scripts" would be better than "bundle".

Reply:
If we do change this to "scripts", we need to change the URL to the JavaScript file in `Views/Layout.cshtml`.